components/BadgeReportStatus.js

```diff
@@ -0,0 +1,43 @@
+import Badge from './Badge'
+import PropTypes from 'prop-types'
+
+const BadgeReportStatus = ({status}) => {
+
+  const allowedStatuses = {
+    0: {
+      name: 'Dikonfirmasi',
+      type: 'danger'
+    },
+    1: {
+      name: 'Ditolak',
+      type: 'secondary'
+    },
+    2: {
+      name: 'Pending',
+      type: 'warning'
+    }
+  }
+
+  function selectedStatus() {
+    return allowedStatuses[status] ?? allowedStatuses[0]
```

Comment:
Default-nya gini aja @rudestewing 

```suggestion
    return allowedStatuses[status] ?? allowedStatuses[2]
```